src/js/books-list.js

```diff
@@ -59,8 +59,7 @@ const allCategory = categoryName => {
 //   };
 // }
 
-const adjustBooksList = () => {
-  booksSection.innerHTML = '';
+const adjustBooksList = () => {;
```

Comment:
Czemu średnik po nawiasie?

Reply:
booksSection.innerHTML = ' ' jest po to żeby wyczyściło wszystko co jest na głównej stronie zanim wejdzie tam nowa rzecz, ale akurat w tym miejscu nie działało jak trzeba i przełożyłem to w inne miejsce

Reply:
OK, rozumiem. Chociaż wygląda jakby został tam średnik w nowej linii nr 62, na końcu - po nawiasie otwierającym. Ale jeśli działa, to spoko ;)

Reply:
działa :)

Reply:
To świetnie :) Dobra robota 😎 Mergować będzie Ania, gdy aktualną wersję sprawdzi Mentor.